### PR TITLE
Fix locally scoped constant array

### PR DESCRIPTION
**Pull request by SimonCropp**
@jbevain seems to me an array should be treated the same as a object type for this context?

### CODE REVIEW

**Comment by SimonCropp** [comment]
put in a diff PR since i am confident the fail test verifies the scenario. but am uncertain about this as a fix

File: Test/Mono.Cecil.Tests/ImageReadTests.cs [hunk 1]
```diff
@@ -207,6 +207,14 @@ public void Net471TargetingAssembly ()
 			});
 		}
 
+		[Test]
+		public void LocallyScopedConstantArray ()
+		{
+			TestModule ("LocallyScopedConstantArray.dll", module => {
+				Assert.IsTrue (module.HasDebugHeader);
```
Comment (jbevain) [inline Test/Mono.Cecil.Tests/ImageReadTests.cs]: We'd need to assert that the constant is indeed null in there.

**Comment by jbevain** [comment]
Thanks for the PR. The fix is good!

File: Test/Mono.Cecil.Tests/ImageReadTests.cs (hunk 1 above)
Comment (SimonCropp) [reply Test/Mono.Cecil.Tests/ImageReadTests.cs]: done

**Comment by jbevain** [comment]
Thanks for the PR!

**Pull request merged**